app/enterprise/0.34-x/developer-portal/how-to/serve-assets/google-cloud-storage.md

```diff
@@ -0,0 +1,77 @@
+---
+title: How to use Google Cloud Storage to serve Kong Dev Portal assets
+toc: false
+---
+
+![Google Cloud Platform](https://konghq.com/wp-content/uploads/2019/03/logo_gcp_horizontal_rgb.png)
+
+### Introduction
+
+**Google Cloud Storage** is a [RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer) [online file storage](https://en.wikipedia.org/wiki/Online_file_storage) 
+[web service](https://en.wikipedia.org/wiki/Web_service) for storing and 
+accessing data on [Google Cloud Platform](https://en.wikipedia.org/wiki/Google_Cloud_Platform) infrastructure. The service 
+combines the performance and scalability of Google's cloud with advanced 
+security and sharing capabilities. It is comparable to [Amazons S3](https://en.wikipedia.org/wiki/Amazon_S3) online 
+storage service.
+
+In this tutorial we will walkthrough how to configure a Google Cloud Storage 
+(GCS) Bucket to serve static content such as images and videos for use in your
+Kong Dev Portal.
+
+
+### Prerequisites
+
+You will need to have the Dev Portal enabled, have editing access to the Dev 
+Portal's template files, and an active Google Cloud Account with billing enabled. 
+
+
+## Step 1 - Create a [Google Cloud Storage (GCS) Bucket](https://console.cloud.google.com/storage/create-bucket)
+
+Navigate to your GCS account and select or create a project to hold your Dev 
+Portal assets. 
+
+Once in your project click the "Storage" option under the "Storage" section in 
```

Comment:
We should definitely adhere to a modern line-level formattion rule for visible interface text, hostnames, terms, and emphasis on context switching.

It makes it easier to parse the document and highlights important details.

See https://www.digitalocean.com/community/tutorials/digitalocean-s-technical-writing-guidelines#formatting